Makefile

```diff
@@ -58,64 +70,72 @@ install-proto-rootgen:
 # -------------------------------
 
 vpp-image:
-	@echo "=> building VPP image -- version=${VPP_VERSION}"
-	IMAGE_TAG=${STONEWORK_VPP_IMAGE} \
-	VERSION=${VPP_VERSION} \
+	@echo "=> building VPP image"
+	VPP_VERSION=${VPP_VERSION} \
+	IMAGE_TAG="${STONEWORK_VPP_IMAGE}:${DEV_VERSION}" \
```

Comment:
This once should be tagged with VPP version, so it follows the same versioning as vpp-base does. No point in tagging vpp image with same tag as stonework.